src/pages/contactUs/components/Form.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Composition, Box } from 'atomic-layout'
+import { GoogleApiWrapper } from 'google-maps-react'
+
+import { Label } from '../atoms/Label'
+import { Button } from '../../../atoms/Button'
+
+const StyledInput = styled.input`
+  background-color: rgba(255, 255, 255, 0.01);
+  border-style: solid;
+  border-width: 0 0 1px;
+  border-color: ${({ theme }) => theme.colors.blackLight};
+  padding: 15px 0 10px;
+  width: 100%;
+`
+
+const StyledTextarea = styled.textarea`
+  color: ${({ theme }) => theme.colors.grey};
+  background-color: rgba(255, 255, 255, 0.01);
+  border-style: solid;
+  border-width: 0 0 1px;
+  border-color: ${({ theme }) => theme.colors.blackLight};
+  padding: 15px 0;
+  width: 100%;
+  height: 120px;
+`
+
+const StyledButton = styled(Button)`
+  background-color: ${({ theme }) => theme.colors.blackLight};
+  border: 2px solid ${({ theme }) => theme.colors.blackLight};
+  font-size: 14px;
+  margin: 30px 0;
+  padding: 18px 28px;
+  max-width: max-content;
+
+  :hover {
+    background-color: ${({ theme }) => theme.colors.white};
+    color: ${({ theme }) => theme.colors.blackLight};
+    border: 2px solid ${({ theme }) => theme.colors.blackLight};
+  }
+`
+
+const StyledSubscribe = styled.span`
+  color: ${({ theme }) => theme.colors.blackLight};
+  margin-left: 5px;
+`
+
+const ContainerCheckbox = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+export const Form: React.FC = () => {
+  const [nameValue, setNameValue] = useState('')
+  const [emailValue, setEmailValue] = useState('')
+  const [messageValue, setMessageValue] = useState('')
+  const [subscribed, setSubscribed] = useState(false)
+
+  const handleNameChange = (event: any) => {
```

Comment:
Maybe let's add proper TypeScript types here instead of `any`? We can discuss this.